cypress/e2e/AdminForm.cy.ts

```diff
@@ -0,0 +1,78 @@
+import { meSuperadmin } from "../intercepts/CreateAdmin";
+
+describe("create admin page", () => {
+  it("shows all elements of the form", () => {
+    meSuperadmin();
+
+    cy.visit("http://localhost:5173/admins/create");
+
+    cy.contains("Create Admin").should("be.visible");
+    cy.contains("Email address").should("be.visible");
+    cy.contains("Password").should("be.visible");
+    cy.contains("Super admin privileges").should("be.visible");
+    cy.get("input[type=checkbox]")
+      .should("be.visible")
+      .should("not.be.checked");
+    cy.contains("button", "Save")
+      .should("be.visible")
+      .should("have.attr", "type", "submit");
+    cy.contains("a", "Cancel")
+      .should("be.visible")
+      .should("have.attr", "href", "/admins");
+  });
+
+  it("creates a new admin", () => {
+    meSuperadmin();
+
+    cy.visit("http://localhost:5173/admins/create");
+
+    cy.get("input[type=email]").type("test@example.com");
+    cy.get("input[type=password]").type("test");
+    cy.get("input[type=checkbox]").click();
+    cy.contains("button", "Save").click();
```

Comment:
Nothing is being tested here aswell. Make sure you add also a method to intercepts for a successfull call (replicate the backend response) and then test the maybe test the redirect and no error message